applications/GeoMechanicsApplication/tests/run_cpp_unit_tests.py

```diff
@@ -1,6 +1,8 @@
 import sys
 import KratosMultiphysics
 from KratosMultiphysics.GeoMechanicsApplication import *
+from KratosMultiphysics.LinearSolversApplication import *
```

Comment:
This import is necessary to initialize all solvers (including e.g. sparse_lu). I get a SonarCloud code smell about this import (https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&branch=geo%2F11626_InitializeSolvingStrategies_Settlement&id=KratosGeomechanics&open=AYsBGTSPtEYIt1WF2mqJ), so let's discuss if that's acceptable.

Reply:
Just a question, wouldn't an `import` like the following also work?
```suggestion
import KratosMultiphysics.LinearSolversApplication
```

Reply:
Seems to work, changed